content/components/dialog.mdx

```diff
@@ -255,6 +255,12 @@ If the answer is yes, designing the feature inside a dialog provides an easy way
   </Dont>
 </DoDontContainer>
 
+### Nested dialogs
+
+It is acceptable to have up to two nested dialogs as long as the dialog is implemented correctly. For example, upon closing the second nested dialog, focus should return to the trigger control for this dialog within the first dialog. When the two stacked dialogs are open, pressing `Esc` should only close the top-level dialog. When one clicks outside of the second dialog with a mouse, only the second dialog should close.
```

Comment:
I think it'd be helpful to link "implemented correctly" to some actionable guidelines: for example, the [ARIA spec](https://www.w3.org/TR/wai-aria-1.2/#dialog) for the `dialog` role or the [Primer Dialog component](https://primer.style/react/Dialog)

Reply:
The Primer React Dialog hasn't been accessibility remediated yet. The ARIA spec doesn't have as much guidance as I'd hoped. Maybe https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/?

Reply:
It's not nested dialog specific guidance though. I feel okay leaving out given we have written out the behavior we're expecting for nested dialogs.

Reply:
This page has a [Trigger elements and keyboard focus](https://primer.style/design/components/dialog#trigger-elements-and-keyboard-focus) section which describes expected behavior. I will link to this.

Reply:
What do you think of https://github.com/primer/design/pull/583/commits/03a3bdf6beeb31fd6e75a8eab920c2052a2f4621?